examples/speaker_recognition/conf/speaker_diarization.yaml

```diff
@@ -15,11 +15,17 @@ diarizer:
     model_path: null #.nemo local model path or pretrained model name or none
     window_length_in_sec: 0.15
     shift_length_in_sec: 0.01
-    threshold: 0.7 # tune threshold on dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
     vad_decision_smoothing: True
     smoothing_params:
       method: "median" 
       overlap: 0.875
+    threshold: null # if not null, use this value and ignore the following onset and offset. Warning! it will be removed in release 1.5!
+    postprocessing_params: # Tuned parameter for CH109! (18 sessions as dev set) You might need to choose resonable threshold or tune the threshold on your dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
+      onset: 0
```

Comment:
shouldn't pad part of this params?

Reply:
We do not carry determined exps for pad_on and pad_off and just use default values 0 at this stage to reduce the number of HP. If they want to use it, they can add them to config with "+"